scripts/preg.py

```diff
@@ -1103,7 +1104,7 @@ def ListDirectoryContent(self, line):
     for key in current_key.GetSubkeys():
       # TODO: move this construction into a separate function in OutputWriter.
       time_string = timelib.Timestamp.CopyToIsoFormat(
-          key.last_written_time)
+          key.last_written_time.GetPlasoTimestamp())
```

Comment:
This method is on the list of deprecation https://github.com/log2timeline/dfdatetime/blob/master/dfdatetime/interface.py#L443